.github/workflows/release.yml

```diff
@@ -156,7 +156,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          persist-credentials: false
+          persist-credentials: true
```

Comment:
is this needed?

Reply:
oh i see to execute the checkout of the stable branch

Reply:
because we are doing git push in later steps we need this credentials to be carried to next step in the current job.

Reply:
yeah that makes sense, i realized after.